src/documentation/pages/components/stat/examples/index.vue

```diff
@@ -0,0 +1,41 @@
+<template lang="pug">
+  div
+    h2 Examples
+
+    div(v-if="$store.state.themes.theme === 'portal'")
+      <BasePortal />
+      <BackgroundIconsPortal />
+      <CompactPortal />
```

Comment:
Could we join compact examples under the same file? They are the same, visually speaking.